wrappers/python/pyrs_device.cpp

```diff
@@ -23,6 +23,8 @@ void init_device(py::module &m) {
         .def("supports", &rs2::device::supports, "Check if specific camera info is supported.", "info"_a)
         .def("get_info", &rs2::device::get_info, "Retrieve camera specific information, "
              "like versions of various internal components", "info"_a)
+        .def("get_saftey_preset_at_index", &rs2::device::get_saftey_preset_at_index, "index"_a)
+        .def("set_saftey_preset_at_index", &rs2::device::set_saftey_preset_at_index, "index"_a, "data"_a)
```

Comment:
Once we will have the sensor extension to `safety_sensor` we will need to add the extension to python as well.
like here:
https://github.com/IntelRealSense/librealsense/blob/12d0a08be7ecd05d66360f5634ec137f8f69e35c/wrappers/python/pyrs_sensor.cpp#L80